x-pack/legacy/plugins/spaces/public/views/space_selector/space_selector.tsx

```diff
@@ -25,7 +26,6 @@ import { Space } from '../../../common/model/space';
 import { SpaceCards } from '../components/space_cards';
 
 interface Props {
-  spaces?: Space[];
```

Comment:
If `this.props.spaces` was provided, it was via injected variables, which have been removed as part of this PR. Now, the selector screen must request the list of spaces on mount.